### DISCUSSION TITLE
List of all auto-imported functions?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [nuxt/nuxt#17920] [Questions] [answered]
Is there a list of all auto-imported functions? I know functions such as `useRoute` are imported automatically in Nuxt 3. Is there a list I can see of all functions that are available?

**Comment by nurdism** [comment, marked as answer]
look at `.nuxt/types/auto-imports.d.ts`

**Reply by CodyBontecou** [reply]
That'll do. Thank you.
